### PR TITLE
CHORES: Prepare 3.2.2

### PR DESCRIPTION
**Pull request by maxulysse**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,13 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## dev
+## [3.2.2](https://github.com/nf-core/sarek/releases/tag/3.2.2) - Pierikjaure
```
Comment (maxulysse) [inline CHANGELOG.md:8]: @asp8200 any idea for a release codename?

Comment (asp8200) [reply CHANGELOG.md:8]: Vuoinesluobbalah?

https://goo.gl/maps/sCnN4XfpTRPxStTd6

Comment (asp8200) [reply CHANGELOG.md:8]: Gällivare?

https://goo.gl/maps/yVERsgxd1tR7VHny9

Comment (maxulysse) [reply CHANGELOG.md:8]: I like `Vuoinesluobbalah`

Comment (maxulysse) [inline CHANGELOG.md:8]:
```suggestion
## [3.2.2](https://github.com/nf-core/sarek/releases/tag/3.2.2) - Vuoinesluobbalah

Vuoinesluobbalah is a lake close to Bierikjávrre.
```

Comment (asp8200) [reply CHANGELOG.md]: It rolls nicely off the tongue

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**